ui/admin/ember-cli-build.js

```diff
@@ -30,6 +30,8 @@ module.exports = function (defaults) {
     app.import('node_modules/clipboard/dist/clipboard.min.js');
   }
 
+  app.import('node_modules/jsonlint/lib/jsonlint.js');
```

Comment:
Is it possible to have this import handled in rose so the consumer doesn't need to to install it?

Reply:
+1 to @ZedLi comment.

I imagine the editor consumption as any other of our Rose components, just using it on the markup.

Reply:
It is possible. Moved this import to rose.